variables.tf

```
@@ -8,6 +8,16 @@ variable "action_targets" {
   default = []
 }
 
+variable "standards_control" {
+  description = "List of Security Hub standards to enable or disable in current region."
+  type = list(object({
+    standards_control_arn = string
+    control_status        = string
+    disabled_reason       = string
+  }))
```

One suggestion. Using for_each over an arn frequently has some odd edge cases, depending on how the module is used. Much prefer to inject a `name` attribute, that the user fully owns and sets up front, even if `name` is required nowhere else except as the for_each label.

```suggestion
  type = list(object({
    name                  = string
    standards_control_arn = string
    control_status        = string
    disabled_reason       = string
  }))
```

The for_each would then be:

```
  for_each = { for control in var.standards_control : control.name => control }
```

---

name isn't a supported attribute for the resource, so trying to inject that just gives me an unsupported argument